packages/quantic/force-app/main/default/lwc/quanticFeedbackModalQna/quanticFeedbackModalQna.js

```diff
@@ -168,7 +168,7 @@ export default class QuanticFeedbackModalQna extends LightningModal {
       return;
     }
     this.handleSubmit({
-      correctTopicValue: this.correctTopicValue,
+      correctTopic: this.correctTopicValue,
```

Comment:
What is that change ? Seems unrelated to me

Reply:
Ah yeah forgot to mention that in pr description,
I also noticed that we were passing a wrong key when passing the parameter to submit feedback from Quantic,
That's the expected type of this parameter:

<img width="1076" alt="Screenshot 2024-11-28 at 11 01 59 AM" src="https://github.com/user-attachments/assets/9456f098-7bb8-4879-b672-459580052324">